### PR TITLE
150 kategorijų sąrašas tituliniame

### LINKED ISSUE #150: Kategorijų sąrašas tituliniame
Modifikuotas "featured kategorijų" sąrašas su paveiksliukais tituliniame lape su galimybe juos įkelti per Django admin, paveiksliukai atvaizduojami su [sorl-thumbnail](https://github.com/jazzband/sorl-thumbnail).  Perkelta Category nuo AL_Node į MP_Node ir parašyta duomenų migraciją.

---

Padaryti kategorijų sąrašą, su paveiksliukais, tokį, koks yra dabar tituliniame lape:

https://data.gov.lt/

- [x] Turėtu būti rodomos tik featured kategorijos.

- [ ] Kategorijos paveiksliukai turi būti įkeliami per Django admin, paveiksliukai atvaizduojami su [sorl-thumbnail](https://github.com/jazzband/sorl-thumbnail).

- [ ] Įkeliant paveiksliukus reikėtų patikrinti ar jų matmenys vienodi.

- [x] Rūšiuojam pagal abėcėlę.

- [x] Dokumentuoti (prie model property help_text) arba įdėti drop down `icon` pasirinkimui.

- [x] Perkelti Category nuo AL_Node į MP_Node ir parašyti duomenų migraciją.

### PR DESCRIPTION
**Pull request by adp-atea**

### CODE REVIEW

File: vitrina/classifiers/models.py [hunk 1]
```diff
@@ -1,27 +1,54 @@
...
+        (AIRPLANE, 'airplane'),
+        (BALANCE, 'balance'),
+        (MARKETING, 'marketing'),
+        (NATURE, 'nature'),
+        (OIL, 'oil'),
+        (PANTHEON, 'pantheon'),
+        (RETIREMENT, 'retirement'),
+        (TRUMPET, 'trumpet'),
+        (VACCINE, 'vaccine'),
+    ]
```
Comment (sirex) [inline vitrina/classifiers/models.py]: Do not hardcode few possible icons, instead provide a link to a full list of icon names.

**Review by sirex — CHANGES_REQUESTED** [review]
Migrations do not work:

```
$ poetry run python manage.py migrate
Applying vitrina_classifiers.0005_auto_20220919_1047...
Traceback (most recent call last):
  File "django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.ObjectInUse: cannot ALTER TABLE "category" because it has pending trigger events
```

File: vitrina/classifiers/models.py (hunk 1 above)
Comment (adp-atea) [reply vitrina/classifiers/models.py]: Currently the icons are all in the icons.css file, I listed them all out in the model, how would I go about automating the process of providing icons from a .css file to the choices field ?

Comment (sirex) [reply vitrina/classifiers/models.py]: Maybe we can use icons as documented in Bulma and provide link to [Font Awesome search page](https://fontawesome.com/search)? This way, when adding a new category, we could pick form this library, by specifying a string like `home`, which would translate into `fa fa home` css class.

Comment (adp-atea) [reply vitrina/classifiers/models.py]: The icons that are used are not part of font awesome, they are custom as I understand. There are 14 of them in total.

Comment (sirex) [reply vitrina/classifiers/models.py]: In this case, could you convert to Bulma icons, which offers wast selection of icons? This way, we will be able to add new categories and assign an icon to it.

For this a migration is needed, that would replace currently used names with alternatives from Bulma icon selection.

**Review by sirex — APPROVED** [review]
Category model migration to MP_Node was completely broken, had to rewrite it completely. Category admin was broken, because of broken migration, category links from landing page were hardcoded and didn't work.

Could you please test if your code works, before submitting for review?

**Pull request merged**